index.bs

```diff
@@ -1875,8 +1875,15 @@ To <dfn>check if cookie-based debugging is allowed</dfn> given a
 
 <h3 algorithm id="obtaining-context-origin">Obtaining context origin</h3>
 
-To obtain the <dfn export for=node>context origin</dfn> of a [=node=] |node|, return |node|'s [=node navigable=]'s
-[=navigable/top-level traversable=]'s [=navigable/active document=]'s [=origin=].
+To obtain the <dfn export for=node>context origin</dfn> of a [=node=] |node|:
+
+1. If applicable for [=implementation-defined=] use cases, return the embedded origin.
+1. Otherwise, return |node|'s [=node navigable=]'s [=navigable/top-level traversable=]'s
+    [=navigable/active document=]'s [=origin=].
+
+Note: The user agent may return the embedded origin to allow separate limits
+for specific use cases. Embedded origin could be the effective top-level origin,
+e.g. the origin of the website displayed in the AMP viewer.
```

Comment:
I still don't think someone reading this is going to understand what an embedded origin is, as it's not a standard term as far as I know. An example would definitely help, but we should also avoid referring to AMP in the spec if possible. Maybe we can give a neutral example like comparing `https://foo.example/` vs `https://foo.example/bar.example`?